script.js

```diff
@@ -299,3 +299,59 @@ workMobile.forEach((work) => {
   });
   mobileWork.appendChild(item);
 });
+
+// form validatio
+
+const form = document.getElementById('form');
+const name = document.getElementById('name');
+const email = document.getElementById('email');
+const message = document.getElementById('message');
+
+function setErrorFor(input, message) {
+  const mess = document.createElement('div');
+  mess.appendChild(document.createTextNode(message));
+  input.parentNode.insertBefore(mess, input.nextElementSibling);
+  mess.style.position = 'absolute';
+  mess.style.color = 'rgb(171 2 2 / 94%)';
+  mess.style.marginBottom = '5px';
+  mess.style.top = `${input.offsetTop + input.offsetHeight}px`;
+  mess.style.left = `${input.offsetLeft}px`;
+  email.style.borderColor = 'rgb(173 25 25 /94%)';
+}
+
+function setSuccessFor(input) {
+  const m = input;
+  m.style.color = 'green';
+}
+
+function checkInputs() {
+  const nameValue = name.value.trim();
+  const emailValue = email.value.trim();
+  const messageValue = message.value.trim();
+
+  if (nameValue === '') {
+    setErrorFor(name, 'Name cannot be blank');
+  } else {
+    setSuccessFor(name);
+  }
+
+  if (emailValue === '') {
+    setErrorFor(email, 'Email cannot be blank');
+  } else if (/[A-Z]/.test(emailValue)) {
+    setErrorFor(email, "Please don't put Uppercase in email !!!");
+    email.style.borderColor = 'rgb(173 25 25 /94%)';
+  } else {
+    setSuccessFor(email);
+  }
+
+  if (messageValue === '') {
+    setErrorFor(message, 'Message cannot be blank');
+  } else {
+    setSuccessFor(message);
+  }
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  checkInputs();
```

Comment:
- [ ] Good job on your validations 👍🏽. I noticed that when I enter the correct details as the form inputs. The form doesn't get submitted. KIndly check this out. 

Attached is a loom video to that effect 👍🏽 
https://www.loom.com/share/092cced7483242ed91284fee8ebfcea5

Reply:
Thank you very much!
I did not notice this problem before.
Now it is fix